params/shapemaker_vhcc_run3/Zll.yaml

```diff
@@ -86,13 +86,13 @@ sample_to_process_map:
   DATA_DoubleMuon: "data_obs"
   ZH_Hto2C_Zto2L: "ZH_hcc"
   ZH_Hto2B_Zto2L: "ZH_hbb"
-  ggZH_Hto2C_Zto2L: "ggZH_hcc"
-  ggZH_Hto2B_Zto2L: "ggZH_hbb"
+  ggZH_Hto2C_Zto2L: "ZH_hcc"
```

Comment:
I don't think we want to merge ggZH with ZH.  For two reasons: 1) at the moment merging is not implemented, they will be overwritten. 2) these are two separate processes and will have different systematics from theory. Even in Run2 they were treated separately.